packages/unlock-js/src/Unlock/v10/createUpgradeableLock.js

```diff
@@ -0,0 +1,109 @@
+import ethersUtils from '../../utils'
```

Comment:
I actually wonder if we should just keep `createLock` that calls the contracts' `createUpgradeableLock`, because I don't think we should allow people to call non upgradable locks?

Reply:
> I think it could be easier to split a bit more because there are a lot of things going on here :/

Yep I splitted it in smaller PRs so its easier to review

Reply:
yep for most part it will be easier to just keep the `createLock` as it is. The main reason why I added the `createUpgradeableLock` is because there is a slight difference : we explicitly pass the `lockOwner` in params, while `createLock` uses `msg.sender`. 

I thought that was enough of a change to add it here - as that logic could be needed. 

We can also remove the `createUpgradeableLock` altogether ofc

Reply:
Ok, let's just remove `createUpgradeableLock`.